lib/Ast_to_Mir.ml

```diff
@@ -305,7 +317,10 @@ let mkdataread id var sizedtype sloc =
   let read_base var =
     { var with
       texpr=
-        FunApp (string_of_internal_fn FnReadData, [mkstring var.texpr_loc id])
+        FunApp
+          ( Mir.StanMath
```

Comment:
I wonder if we should just go ahead and have a separate funkind for these internal functions? That could be handy for switching on later, actually.